static/js/src/certification-results.js

```diff
@@ -0,0 +1,142 @@
+        if (releaseChipValue) {
+          if (releaseFilterCount === 0) {
+            chip.classList.toggle("u-hide");
+          }
+          releaseChipValue.innerHTML = releaseFilterCount;
+        }
+
+        // Add line break in title to keep chip on same line
+        const header = tab.querySelector(".p-certification-header");
+        if (header.innerHTML.includes("Certified")) {
+          if (!chip.classList.contains("u-hide")) {
+            header.innerHTML = "Certified Ubuntu <br> release";
+          } else {
+            header.innerHTML = "Certified Ubuntu release";
+          }
+        }
+      });
+    });
+  }
+  displayFilterCount();
+}
+
+enableApplyFilters();
+
+//function to ensure only the option which has been changed is appended to the URL
+function updateResultsPerPage() {
+  let resultsDropdowns = document.querySelectorAll(".p-results-per-page");
+  resultsDropdowns.forEach((resultsDropdown) => {
+    const options = resultsDropdown.querySelectorAll("option");
+    options.forEach((option) => {
+      if (option.selected) {
+        if (option.value === resultsDropdown.dataset.limit) {
+          resultsDropdown.name = "";
+        }
+        searchResults.submit();
+      }
+    });
+  });
+}
+
+function toggleVendorsList() {
+  const vendorSection = document.querySelector("#vendor-section");
+  const vendorPanel = vendorSection.querySelector(".p-accordion__panel");
+  const toggleVendorButtons = document.querySelectorAll(".js-reveal-vendors");
+  toggleVendorButtons.forEach((button) => {
+    button.addEventListener("click", () => {
+      window.scrollTo(0, 360);
+      vendorPanel.classList.toggle("is-collapsed__vendors");
+      toggleVendorButtons.forEach((button) => {
+        button.classList.toggle("u-hide");
+      });
+    });
+  });
+}
+toggleVendorsList();
+
+function toggleVersionsList() {
+  const versionSection = document.querySelector("#version-section");
+  const versionPanel = versionSection.querySelector(".p-accordion__panel");
+  const toggleVersionButtons = document.querySelectorAll(".js-reveal-versions");
+  toggleVersionButtons.forEach((button) => {
+    button.addEventListener("click", () => {
+      versionPanel.classList.toggle("is-collapsed__versions");
+      toggleVersionButtons.forEach((button) => {
+        button.classList.toggle("u-hide");
+      });
+    });
+  });
+}
+toggleVersionsList();
+
+//hides "show all .." and "show fewer" links when the accordion is closed
+function toggleShowAllLinks() {
+  const filterSections = document.querySelectorAll(
+    ".p-accordion__group--certified"
+  );
+  filterSections.forEach((filterSection) => {
+    const tab = filterSection.querySelector(".p-accordion__tab");
+    tab.addEventListener("click", () => {
+      const links = filterSection.querySelector(".js-toggle-links");
+      if (links) {
+        links.classList.toggle("u-hide");
+      }
+    });
+  });
+}
+toggleShowAllLinks();
```

Comment:
I think maybe you should stack all this functions calls together?, so we can clearly see at one glance which functions are being executed in this script.
```
...
toggleVendorsList();
toggleShowAllLinks();
toggleVersionsList();
```